low-level/sspl-ll.spec

```diff
@@ -94,9 +95,9 @@ SSPL_DIR=/opt/seagate/%{product_family}/sspl
     ln -sf $SSPL_DIR/low-level/consuldump.py $SSPL_DIR/bin/consuldump.py
     ln -sf $SSPL_DIR/low-level/resource_health_view /usr/bin/resource_health_view
     ln -sf $SSPL_DIR/low-level/sspl_bundle_generate /usr/bin/sspl_bundle_generate
-    ln -sf $SSPL_DIR/low-level/manifest_support_bundle /usr/bin/manifest_support_bundle
     ln -sf $SSPL_DIR/low-level/framework $SSPL_DIR/low-level/solution/lr2/
     ln -sf $SSPL_DIR/low-level/solution $SSPL_DIR/extension/solution
+    ln -sf $SSPL_DIR/low-level/sspl_support_bundle $SSPL_DIR/bin/sspl_support_bundle
```

Comment:
can stop soft link creation for cortxv1 sb script now, i.e
sspl_bundle_generate
manifest_support_bundle
@ line 92, 93, 131, as it would confuse otherwise

Reply:
Done